vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -212,7 +212,9 @@ function Test-ImagesPulled {
         $pulledImages = (ctr.exe -n k8s.io image ls -q | Select-String -notmatch "sha256:.*" | % { $_.Line } )
     }
     elseif ($containerRuntime -eq 'docker') {
-        $pulledImages = docker images --format "{{.Repository}}:{{.Tag}}"
+        Start-Service docker
```

Comment:
Does it `Start-Service docker` work?

Reply:
This is the way we used in https://github.com/Azure/AgentBaker/blob/6618e34f10c78c38bbd02fc357bb4e1c2cb155e2/vhdbuilder/packer/configure-windows-vhd.ps1#L222-L231

Reply:
in test

Reply:
Tried 10 times, all tests passed. Could you please take a look?
https://msazure.visualstudio.com/CloudNativeCompute/_build?definitionId=182855&_a=summary